sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClient.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.annotation.ServiceClient;
+
+import java.util.List;
+
+@ServiceClient(
+    builder = TableServiceClientBuilder.class)
+public class TableServiceClient {
+
+    TableServiceClient() {
+    }
+
+    /**
+     * creates the table with the given name.  If a table with the same name already exists, the operation fails.
+     *
+     * @param name the name of the table to create
+     */
+    public AzureTable createTable(String name) {
+        return null;
+    }
+
+    /**
+     * deletes the given table. Will error if the table doesn't exists or cannot be found with the given name.
+     *
+     * @param name
+     */
+    public void deleteTable(String name) {
```

Comment:
Should we have a version of this method that takes an `AzureTable` object as well? Not sure myself, open for debate.